c7n/resources/awslambda.py

```diff
@@ -255,6 +255,45 @@ class KmsFilter(KmsRelatedFilter):
     RelatedIdsExpression = 'KMSKeyArn'
 
 
+@AWSLambda.action_registry.register('enable-xray-tracing')
```

Comment:
let's change this to `set-xray-tracing` and have a `state` key for true/false so we can enable and disable

Reply:
Updated action with state key.